### PR TITLE
Write deps to bower.json

### PR DESCRIPTION
**Pull request by jimmynotjim**
Writing CF component dependencies to the bower file

## Additions

None

## Removals

None

## Changes

- updated index.js to get latest release from GitHub api
- updated `_bower.json` template to write user chosen deps

## Testing

- fetch and run `npm link`
- run `mocha`
- check /test/temp/bower.json for list of deps with version numbers

## Review

- @contolini 
- @ascott1 
- @Scotchester 
- @KimberlyMunoz
- @himedlooff 

## Preview

screen shot 2015-03-25 at 2 46 59 pm

## Notes

- pings GitHub tags endpoint and grabs most recent release tag
- sets the repo name and version in bower template
- waits to call tag endpoint until user has set preferred components to avoid erroneous API calls
- Related to #30

### CODE REVIEW

**Comment by jimmynotjim** [comment]
I added an API call to the tags endpoint for each chosen component and updated the `bower.json` for the new `this.components` array. Not sure if this is the cleanest solution, but it's the only way I could find that avoided erroneous API calls and worked with async instead of against it.

**There are now conflicts from the other work today, once this is OK'd I'll squash everything and then pull down the upstream changes

File: app/index.js [hunk 1]
```diff
@@ -118,7 +118,22 @@ var CapitalFrameworkGenerator = yeoman.generators.Base.extend({
...
-          this.components = answers.components;
+          var versionedComponents = this.components = [];
+
+          // Get latest repo tag
+          var getLatest = function( repo ) {
+            request({
+              uri: 'https://api.github.com/repos/cfpb/' + repo + '/tags',
+              json: true,
+              headers: { 'user-agent': 'generator-cf'}
+            },function(err,res, data) {
```
Comment (anselmbradford) [inline app/index.js]: Minor:
`}, function( err, res, data ) {`

Comment (jimmynotjim) [inline app/index.js]: Fixed in 988adf2

**Comment by Scotchester** [comment]
Looks good to me. Squash it up!

**Comment by Scotchester** [comment]
Ready for merge?

**Comment by jimmynotjim** [comment]
Yes it is, thanks.

**Pull request merged**